### PR TITLE
feat(winc for token): init getWincForToken PE-6632

### PR DESCRIPTION
**Pull request by fedellen**

### CODE REVIEW

**Comment by fedellen** [comment]
not yet passing CI. will continue tomorrow

File: src/types.ts [hunk 1]
```diff
@@ -84,12 +84,33 @@ export type TurboWincForFiatResponse = TurboPriceResponse & {
...
 export type TurboWincForFiatParams = {
   amount: CurrencyMap;
   nativeAddress?: NativeAddress;
   promoCodes?: string[];
 };
 
+export type TurboWincForTokenParams = {
+  tokenAmount: BigNumber.Value;
+  // nativeAddress?: NativeAddress; TODO: Crypto promo code support on payment service
+  // promoCodes?: string[]; TODO: Crypto promo code support on payment service
```
Comment (dtfiedler) [inline src/types.ts:111]: remove?

Comment (fedellen) [reply src/types.ts]: 5870066

**Pull request merged**